docker/README.md

```diff
@@ -32,6 +32,17 @@ copy the certificate to `nginx-plus-server/nginx-repo.crt` and the key to `nginx
 Another option is to change the paths to the certificate and key files in lines 12 and 13 of
 `nginx-plus-server/Dockerfile`.
 
+## Run with the legacy NGINX Plus API
+
+Current `nginx-plus-server/Dockerfile` is configured to run the NGINX+ with the `newer` API. In order to run
```

Comment:
nit: using `legacy` and `newer` as reserved or quoted words is no longer relevant (throughout).

Reply:
As per the current implementation, we have removed the quotes from the words legacy and newer.